features/account/accesibility.feature

```diff
@@ -0,0 +1,21 @@
+Feature: User settings page
+    As a logged user, in any page
+    I want to toggle Use High Contrast UI from The left menu in the account section
+    So I can view page more easily
```

Comment:
pages

Reply:
@ORIGIN-Ticket-008
Feature: Accesibility
As a logged user, on any page, I want to be able to toggle Use High Contrast UI  from the left menu in the Account section. So the visibility for the user is way better.